docs/src/pages/[platform]/getting-started/installation/dependencies.react.mdx

```diff
@@ -36,19 +36,19 @@ export default function App() {
 
 ### Custom Theme
 
-To add a [custom theme](/theming), wrap your app in the `AmplifyProvider` component and include the default styling.
+To add a [custom theme](/theming), wrap your app in the `ThemeProvider` component and include the default styling.
```

Comment:
Just a note, when you merge in latest from main,  this section was removed since it doesn't provide much information about how to actually add a custom theme. We can probably keep it removed?

Reply:
thanks this has been removed